src/test/java/testresults/PostTestResultsNegTestTypesCancelledLvl.java

```diff
@@ -118,6 +120,7 @@ public void testResultsExtraFieldTestExpiryDate() {
         testResultsSteps.validatePostErrorData("testExpiryDate", "is not allowed");
     }
 
+    @Ignore("blocked by CVSB-9639")
```

Comment:
defect was fixed, please remove the @ignore annotation

Reply:
Removed.